app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  before_action :logged_in_user, only: [:create]
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user_id = current_user.id
+
+    if @comment.save
+      respond_to do |format|
+        format.js
+      end
+    else
+      redirect_to posts_path
+    end
+  end
+
+  private
+
+    def comment_params
+      params.require(:comment).permit(:content)
+    end
+end
```

Comment:
You can define a private method call `prepare_post` and add a filter for some methods, such as: `create, show, edit, ....`. In `prepare_post` you find a post via `id` and assign it to `@post`.